data/faq/writing_a_great_proposal.md

```diff
@@ -14,3 +14,23 @@ We want this to be a fun and educating experience for you!
 
 Use the"draft" feature on the GSoC system, submit your draft early, get a
 lot of helpful feedback from us!
+
+
+#### Writing the proposal
+
+Please follow the **proposal template** at [PSF-ApplicationTemplate2016](https://wiki.python.org/moin/SummerOfCode/ApplicationTemplate2016). All fields are required if not explicitly indicated otherwise. While following the application template you can consider doing an illustration of your proposed solution as it makes the application much more intuitive.
+
+For writing your proposal we recommend using **Google Docs** https://www.google.com/docs/about/.
```

Comment:
I'd make google docs a link instead of fat and then the URL, also add something like `since this will be supported by the Google submission system in the end.`